en/setup/config_initial.md

```diff
@@ -10,8 +10,8 @@ The equipment below is highly recommended:
 
 * A Taranis Plus remote control for the safety pilot (or equivalent)
 * A development computer:
-  * MacBook Pro (early 2015 and later) with OSX 10.13 or later
-  * Lenovo Thinkpad 450 (i5) with Ubuntu Linux 16.04 or later
+  * MacBook Pro (early 2015 and later) with OSX 10.13 or later 
```

Comment:
@julianoes Does any of the rest of this need to be changed?

Reply:
Should read OS X 10.15 or later